users/tests/test_user_signup.py

```diff
@@ -10,6 +11,7 @@
 class SignUpTests(APITestCase):
```

Comment:
You should add a unit test to check this:
a new user singup
check that the user has been created in the db but he can't login
confirm the account
check that the user can login